test/WebJobs.Script.Tests/Configuration/DefaultDependencyValidatorTests.cs

```diff
@@ -87,7 +87,7 @@ private async Task<LogMessage> RunTest(Action<IServiceCollection> configureWebHo
                         }));
 
                         string uniqueTestRootPath = Path.Combine(Path.GetTempPath(), "FunctionsTest", "DependencyValidatorTests");
-
+                        Directory.CreateDirectory(uniqueTestRootPath);
```

Comment:
Why is this change required?

Reply:
As mentioned in the comment below. The tests did not expect the host to read the function metadata before. Because of this it did not matter if the rootpath existed or not. But now that we have moved up the logic to read the metadata the tests invariably do need the path to be present.